src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -367,11 +367,11 @@ DictionaryModal.propTypes = {
   buttonname: PropTypes.string,
   fetchingOrganizations: PropTypes.func.isRequired,
   submit: PropTypes.func.isRequired,
-  organizations: PropTypes.object.isRequired,
+  organizations: PropTypes.array.isRequired,
```

Comment:
Please define the array shape. Visit [here](https://til.hashrocket.com/posts/ytrzhrpfxk-proptypes-array-of-shape) for more context.